### PR TITLE
Moved: ValveCorporation.SteamLink to Valve.SteamLink

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103269)

### CODE REVIEW

**Comment by kbdharun** [comment]
@stephengillie I have a doubt, Since Steam and Steam Link are published under ValveCorporation and not just Valve, I moved the manifest to the latter and updated it. Is this anything else, I must do in this PR to fix the Validation warning? Or is it not possible to change the publisher at all?

**Review by russellbanks — CHANGES_REQUESTED** [review]
Addition and deletion are only allowed in the same directory. This would have to be two PRs. I'll wait on another moderator though for whether this should be moved

**Comment by kbdharun** [comment]
> Addition and deletion are only allowed in the same directory. This would have to be two PRs. I'll wait on another moderator though for whether this should be moved

If required, I will separate the move and change to separate PRs. I don't know if this change will break existing users.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Since Valve.Steam is a popular (I guess?) package and the fact it was added earlier in the repo, I would advise against moving this package to avoid breaking any existing scripts that people may be using to install this package.

I would much rather prefer if we moved `ValveCorporation.SteamLink` into `Valve.SteamLink` and avoid duplicate directories

**Comment by SpecterShell** [comment]
Short publisher name is preferred, for example `Microsoft.*` instead of `MicrosoftCorporation.*`

**Comment by kbdharun** [comment]
> Short publisher name is preferred, for example `Microsoft.*` instead of `MicrosoftCorporation.*`

Noted, I will move the Steam link to the shorter Valve directory later today.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Only one manifest can be modified per PR. You'll have to revert the changes you've made to `Valve.Steam`

For the move process, you'll have to create two separate PRs, one for deleting `ValveCorportation.SteamLink` and the other for adding `Valve.SteamLink`. If you're confused, you can take a look at this example

- Deletion PR https://github.com/microsoft/winget-pkgs/pull/101303
- Addition PR https://github.com/microsoft/winget-pkgs/pull/101305

**Comment by stephengillie** [comment]
Automatic Validation failed with:
>  The pull request contains more than one manifest. Either singleton or multi file manifest under one version directory.

**Comment by kbdharun** [comment]
> For the move process, you'll have to create two separate PRs, one for deleting `ValveCorportation.SteamLink` and the other for adding `Valve.SteamLink`. If you're confused, you can take a look at this example

Thanks for the heads up, I have separated the changes into 3 separate PRs, the other 2 being 
- https://github.com/microsoft/winget-pkgs/pull/103450 and 
- https://github.com/microsoft/winget-pkgs/pull/103451.

Sorry for the delay 😄 (I had some work).

**Comment by mdanish-kh** [comment]
The other two PRs look good, this one needs a tiny change to pass the pipelines:

`ValveCorporation.SteamLink` was deleted in this PR/branch, you'll have to revert this change and put it back in it's folder as the deletion is handled in separate PR #103450

**Comment by mdanish-kh** [comment]
Or update your branch and merge changes from the master branch as the deletion PR is now merged

**Comment by kbdharun** [comment]
> ...and you beat me to it 😅

Lol

File: manifests/v/Valve/SteamLink/1.2.0/Valve.SteamLink.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Valve.SteamLink
+PackageVersion: 1.2.0
+PackageLocale: en-US
+Publisher: Valve Corporation
+PackageName: Steam Link
```
Comment (mdanish-kh) [inline manifests/v/Valve/SteamLink/1.2.0/Valve.SteamLink.locale.en-US.yaml:8]:
```suggestion
PackageName: Steam Link
PackageUrl: https://store.steampowered.com/app/353380/Steam_Link/
```
Additional metadata

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/v/Valve/SteamLink/1.2.0/Valve.SteamLink.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: msi
+  NestedInstallerFiles:
+  - RelativeFilePath: SteamLink.msi
+  InstallerUrl: https://media.steampowered.com/steamlink/windows/latest/SteamLink.zip
+  InstallerSha256: C484EEE12470ACCA5437A4AF259E721DB84A6C9AD22C70D569A05D73F5A64C43
+  ProductCode: '{BF6669C3-9C1D-4D0C-9468-38A91F6BA95B}'
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply manifests/v/Valve/SteamLink/1.2.0/Valve.SteamLink.installer.yaml]: Updated the SHA, there is no change in product code, so left it as is.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**